### PR TITLE
Remove unnecessary callback; add notice if plugin is already setup

### PR DESCRIPTION
**Pull request by joshcanhelp**
### Changes

- Add a notice on the Setup Wizard and remove the link from the menu if the plugin has already been configured
- Remove `wp-login.php` from callbacks (not used anymore)

### CODE REVIEW

File: lib/WP_Auth0_Settings_Section.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function init_menu() {
 
 			add_submenu_page( $main_menu, __( 'Help', 'wp-auth0' ), __( 'Help', 'wp-auth0' ), 'manage_options', 'wpa0-help', array( $this, 'redirect_to_help' ) );
 
-			add_submenu_page( $main_menu, __( 'Auth0 for WordPress - Setup Wizard', 'wp-auth0' ), __( 'Setup Wizard', 'wp-auth0' ), 'manage_options', 'wpa0-setup', array( $this->initial_setup, 'render_setup_page' ) );
+			add_submenu_page( null, __( 'Auth0 for WordPress - Setup Wizard', 'wp-auth0' ), __( 'Setup Wizard', 'wp-auth0' ), 'manage_options', 'wpa0-setup', array( $this->initial_setup, 'render_setup_page' ) );
```
Comment (joshcanhelp) [inline lib/WP_Auth0_Settings_Section.php]: Removes the menu item when the plugin is already setup.

File: lib/initial-setup/WP_Auth0_InitialSetup_ConnectionProfile.php [hunk 2]
```diff
@@ -11,7 +11,6 @@ public function __construct( WP_Auth0_Options $a0_options ) {
 	}
 
 	public function render( $step ) {
-		$scopes = WP_Auth0_Api_Client::GetConsentScopestoShow();
```
Comment (joshcanhelp) [inline lib/initial-setup/WP_Auth0_InitialSetup_ConnectionProfile.php]: Moved to `templates/initial-setup/connection_profile.php`

File: lib/WP_Auth0_Settings_Section.php (hunk 1 above)
Comment (cocojoe) [reply lib/WP_Auth0_Settings_Section.php]: But it's always `null` how can it be toggled ?

Comment (joshcanhelp) [reply lib/WP_Auth0_Settings_Section.php]: This is in an `if` block that checks if the plugin is configured or not. If it's already configured, we don't want to show the wizard.

Comment (cocojoe) [reply lib/WP_Auth0_Settings_Section.php]: ok expanded the view, so if it's not ready I can see it adds the `Setup Wizard`. 
So it must be already added via WP_Auth0_Admin.php? Hence the requirement to removal?

**Comment by cocojoe** [comment]
@joshcanhelp you should modify your codecov to allow a little leeway. Take a look at https://github.com/auth0/Auth0.swift/blob/master/codecov.yml

**Comment by joshcanhelp** [comment]
@cocojoe - Well, I am adding a fair bit of untested code. This isn't really the type that would benefit from a test but still, I see CodeCov's point :)

**Comment by cocojoe** [comment]
@joshcanhelp  Yeah that's why you need a little tolerance in the patch/project % values, sometimes it's just not possible and I don't want to end up having to force merge everytime 😄

**Comment by joshcanhelp** [comment]
@cocojoe - I can rebase and merge here without the check, just need an approval.

**Pull request merged**